src/Traduora/TraduoraClient.cs

```diff
@@ -1,38 +1,70 @@
-﻿using System.Net.Http;
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Net.Http;
 using System.Threading.Tasks;
 using Newtonsoft.Json.Linq;
 using Refit;
-using Traduora.Client.Api;
+using Traduora.Provider.Api;
 
-namespace Traduora.Client
+namespace Traduora.Provider
 {
     public class TraduoraClient
     {
-        private readonly HttpClient _httpClient;
+        private readonly ITraduoraApi _traduoraApi;
         private const string DefaultFormat = "jsonnested";
 
         public TraduoraClient(HttpClient httpClient)
         {
-            _httpClient = httpClient;
+            _traduoraApi = RestService.For<ITraduoraApi>(httpClient);
         }
 
-        public async Task<JObject> GetTranslations(string projectId, string locale, string auth,
+        public async Task<string> Authenticate(string clientId, string clientSecret)
+        {
+            var authBody = new ApiCredentials { ClientId = clientId, ClientSecret = clientSecret };
+
+            JToken responseJson = await _traduoraApi.GetToken(authBody);
+
+            return responseJson["data"]["accessToken"].Value<string>();
+        }
+
+        public async Task<IReadOnlyDictionary<string, IReadOnlyDictionary<string, string>>>
+            GetData(string projectId, string authToken,
             string format = DefaultFormat)
         {
-            var traduoraApi = RestService.For<ITraduoraApi>(_httpClient);
+            var output = new Dictionary<string, IReadOnlyDictionary<string, string>>();
+
+            JObject localesJson = await _traduoraApi.GetLocales(projectId, $"Bearer {authToken}");
```

Comment:
Create a model that can be deserialized to instead of JObject/JToken/JWhatever. Would make code simpler and typed.